index.html

```diff
@@ -64,36 +77,53 @@ <h3 class="Proj2">Multi-Post Stories</h3>
                     <li class="Java1"><a href="#">Javascript</a></li>
                 </ul>
                 <a href="#" class="button1">See Project</a>
+                </div>
             </div>
 
             <div class="container3">
                 <div class="rect3"></div>
-                <h4 class="Proj3">Tonic</h4>
+                <div class="left_block2">
+                <h4 class="Proj3a">Tonic</h4>
+                <h4 class="Proj3">Facebook 360</h4> 
+                <ul class="list1a">
+                <li class="frame2a"><img src="Images/Frame2.png" alt="frame"></li>
+                </ul>
+                <p class="text1a">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```

Comment:
- [optional ] Great job making the changes. Please fix this extra right margin in both your mobile and desktop right margin
![Screenshot (345)_LI](https://user-images.githubusercontent.com/65298048/186992515-7f5739c1-40a8-4869-9a03-79c6f2da6e7b.jpg)
![Screenshot (347)_LI](https://user-images.githubusercontent.com/65298048/186992618-5fa765a1-4319-402c-8a19-55cbc21921f9.jpg)